de-lib/src/main/java/org/iplantc/de/apps/client/views/CyVerseReactComponents.java

```diff
@@ -0,0 +1,21 @@
+package org.iplantc.de.apps.client.views;
+
+import org.iplantc.de.apps.client.AppDetailsView;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+
+/**
+ * @author psarando
+ */
+@JsType(isNative = true, namespace = JsPackage.GLOBAL)
+public final class CyVerseReactComponents {
```

Comment:
The methods in this `CyVerseReactComponents` class replace the native methods in `AppDetailsViewImpl` that were calling the React lib functions.